gerrychain/partition/partition.py

```diff
@@ -41,6 +41,10 @@ def _first_time(self, graph, assignment, updaters):
 
         self.assignment = get_assignment(assignment, graph)
 
+         if not self.validate_assignment():
```

Comment:
Looks like there's an extra space before this `if` that's causing the failing tests. You can fix it manually or outsource it to a formatter like [black](https://black.readthedocs.io/en/stable/).

Reply:
edited manually